web/sharings/sharings.go

```diff
@@ -363,7 +363,21 @@ func AddRecipientsDelegated(c echo.Context) error {
 				} else {
 					states[email] = state
 				}
+
+				// If we have an URL for the Cozy, we can create a shortcut as an invitation
+				if cozy != "" {
```

Comment:
nit: Maybe we should move this block before updating `states` to avoid overriding `states[""]` in case we have multiple recipients without emails.
It might not be used today but it could become the source of bugs if we ever care about it.

We can even move the block before even trying to get the state?!

Reply:
> Maybe we should move this block before updating states to avoid overriding states[""] in case we have multiple recipients without emails.

It doesn't change anything, it would still update the `states` after the block (no return for the success case). And, we are using the instance URL as key when the URL is missing, so it's not an issue when there are multiple recipients without emails.